### PR TITLE
Chore/update dependencies

### PR DESCRIPTION
**Pull request by vvandijck**
This pull-request includes:
- Updated dependencies
- Fixed any issues after dependencies were updated.

### CODE REVIEW

File: lib/data-loader/dump-data.js [hunk 1]
```diff
@@ -20,7 +11,6 @@ require('dotenv').config()
 const baseDir = path.join(__dirname, '..', '..')
 const client = new SiteClient(process.env.DATO_API_TOKEN, { 'X-Reason': 'dump', 'X-SSG': '' })
 const configFile = path.join(baseDir, 'dato.config.js')
-const destinationPath = dumpRoot
 const isPreviewMode = /^(?:staging|development)$/.test(process.env.ENVIRONMENT)
 
-module.exports = () => dump(configFile, client, isPreviewMode, destinationPath)
+module.exports = () => Promise.resolve(configFile, client, isPreviewMode, dumpRoot)
```
Comment (thadeetrompetter) [inline lib/data-loader/dump-data.js]: waarom doe je dit? er wordt nu geen dato content gedumpt, lijkt het?

Comment (vvandijck) [reply lib/data-loader/dump-data.js]: `dumpData()` on line 6 in `bin/dump-data.js` expects it to be a `Promise`

Comment (thadeetrompetter) [reply lib/data-loader/dump-data.js]: that promise is supposed to be returned from `dump`. now nothing is being dumped

File: package.json [hunk 2]
```diff
@@ -32,54 +32,54 @@
...
-    "fontfaceobserver": "^2.0.13",
-    "globby": "^8.0.1",
-    "gsap": "^2.0.1",
-    "helmet": "^3.8.1",
-    "isomorphic-fetch": "^2.2.1",
-    "next": "^8.0.4",
-    "lodash": "^4.17.11",
+    "axios": "0.18.0",
+    "backoff": "2.5.0",
+    "cheerio": "1.0.0-rc.3",
```
Comment (Siilwyn) [inline package.json]: If you use a package-lock please add caret ranges to all versions. ^^

**Pull request merged**